src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import Header from "../../base/header/Header";
+import Footer from "../../base/footer/Footer";
+import { Button } from "../../base/button/Button";
+import SideCards from "../../composite/sideCards/SideCards";
+
+import "../page.css";
+import "./BcscRedirect.css";
+
+export default function BcscRedirect({ page: { header } }) {
+  const loginBtn = {
+    label: "LOGIN",
+    buttonStyle: "btn ecrc_common_btn",
+    buttonSize: "btn",
+    type: "button"
+  };
+
+  const accountBtn = {
+    label: "SET UP ACCOUNT",
+    buttonStyle: "btn ecrc_common_btn",
+    buttonSize: "btn",
+    type: "button"
+  };
+
+  const requestBtn = {
+    label: "REQUEST FORM",
+    buttonStyle: "btn ecrc_common_btn",
+    buttonSize: "btn",
+    type: "button"
+  };
+
+  const onLoginClick = () => {
+    window.open("https://support.wwf.org.uk", "_blank");
```

Comment:
what does `_blank` do in this function?

Reply:
_blank will start a new browser tab.
That is the behaviour in mock page